net-p2p/syncthing/files/syncthing-relaysrv.initd

```diff
@@ -0,0 +1,24 @@
+#!/sbin/openrc-run
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+SR_USER=syncthing-relaysrv
+SR_GROUP=syncthing-relaysrv
+SR_HOMEDIR=/var/lib/syncthing-relaysrv
+
+description="Relay service for syncthing"
+command="/usr/bin/relaysrv"
+command_args="${SR_OPTS}"
+pidfile="/run/relaysrv.pid"
+start_stop_daemon_args="--background
+  --user ${SR_USER}
+  --group ${SR_GROUP}
+  --chdir \"${SR_HOMEDIR}\"
+  --make-pidfile "
```

Comment:
Yes. I could not figure out how to log properly. Mainly because there is no mechanism to support logrotate, except restarting the service. Restarting the service seems like a bad idea, because of the web site http://relays.syncthing.net/ where one can get a growing bubble corresponding to the amount of transferred data, and this bubble would reset :-( Also, restarts would affect users of the relay.